### PR TITLE
Submit form, transition to show

### PR DESCRIPTION
**Pull request by allthesignals**
Adds submit button. Cleans up show template

### CODE REVIEW

File: client/app/components/packages/pas-form/edit.js [hunk 1]
```diff
@@ -14,6 +17,13 @@ export default class PasFormComponent extends Component {
     packageInstance.saveDescendants();
   }
 
+  @action
+  async submit(packageInstance) {
```
Comment (godfreyyeung) [inline client/app/components/packages/pas-form/edit.js:21]: We've been using `projectPackage`. Couldn't find any existing instances of `packageInstance` as a parameter name

Comment (allthesignals) [reply client/app/components/packages/pas-form/edit.js:21]: okay will change

Comment (godfreyyeung) [reply client/app/components/packages/pas-form/edit.js:21]: okay, i lied, just one instance in the `edit.js` file.

but there's 47 uses of projectPackage.

this is a super nitpick :p

Comment (allthesignals) [reply client/app/components/packages/pas-form/edit.js]: nope you're right, i didn't like the naming there either. we know it's an instance anyways!

**Pull request merged**